pinot-spi/src/main/java/org/apache/pinot/spi/env/CommonsConfigurationUtils.java

```diff
@@ -101,10 +101,16 @@ private static Iterable<String> getIterable(Iterator<String> keys) {
    * @param configuration to iterate on keys
    * @return a stream of keys
    */
+  @Deprecated
   public static Stream<String> getKeysStream(Configuration configuration) {
     return StreamSupport.stream(getIterable(configuration.getKeys()).spliterator(), false);
   }
 
+  public static Stream<String> getKeysStream(org.apache.commons.configuration2.Configuration configuration) {
```

Comment:
Put `org.apache.commons.configuration2.Configuration` to import section and make the deprecated `org.apache.commons.configuration.Configuration` to full name.